shipstation_integration/shipments.py

```diff
@@ -158,3 +184,57 @@ def create_delivery_note(shipment, sales_invoice):
 	dn.submit()
 	frappe.db.commit()
 	return dn
+
+
+def create_shipment(
+	shipment: "ShipStationOrder",
+	delivery_note: "DeliveryNote",
+	store: "ShipstationStore",
+):
+	shipment_doc: "Shipment" = make_shipment(delivery_note.name)
+	shipment_doc.update(
+		{
+			# todo: change description of content to shipment items.
+			"shipment_id": shipment.shipment_id,
+			"pickup_date": shipment.create_date,
+			"carrier": shipment.carrier_code,
+			"carrier_service": shipment.service_code,
+			"awb_number": shipment.tracking_number,
+			"service_provider": "Shipstation",
+			"incoterm": "DAP (Delivered At Place)",
+			"shipstation_store_name": store.store_name,
+			"shipstation_order_id": shipment.order_id,
+			"marketplace": store.marketplace_name,
+			"marketplace_order_id": shipment.order_number,
+		}
+	)
+
+
+	if shipment.shipment_items:
+		shipment_items = [shipment_item.get("name") for shipment_item in shipment.get("shipment_items")]
+		shipment_qty = [shipment_item.get("quantity") for shipment_item in shipment.get("shipment_items")]
+		description_of_content = {shipment_items[i]: "Qty " + shipment_qty[i] for i in range(len(shipment_items))}
```

Comment:
I forgot to mention it in the other comment, but this should show stock quantity and stock UOM. I think a for loop might be better here instead of list-comp

For future ref, if you're trying to combine lists and iterate over them, you can use Python's [`zip`](https://www.programiz.com/python-programming/methods/built-in/zip) in-built.